tscached.yaml

```diff
@@ -27,8 +27,9 @@ tscached:
         staleness_threshold: 10  # data up to this far in the past is "new"
 
     chunking:
-        chunk_length: 1800  # chunk on 30 minute intervals
+        chunk_length: 3600  # chunk on 1 hour intervals
```

Comment:
Do we want to update the default in "get_chunked_time_ranges"? Currently it defaults to 1800 if no config is defined.